lib/transforms/analyzer.cc

```diff
@@ -104,6 +103,9 @@ static void RunOnInstruction(Instruction* inst,
     case OpCode::CMP: {
       auto& node_info = GenerateCommonInfo(inst, node_infos);
       node_info.flops = inst->GetResultType().GetTotalNumOfElements();
+      DefaultDataLayout dl;
```

Comment:
we can define "dl" as global static.

Reply:
defined a static global DataLayout pointer.